docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md

```diff
@@ -42,3 +42,13 @@ validate remote reports from enclaves.
 ```cmd
 cpack -D CPACK_NUGET_COMPONENT_INSTALL=ON -DCPACK_COMPONENTS_ALL=OEHOSTVERIFY
 ```
+
+## Install the NuGet package
+
+Before the `.nupkg` packages can be installed, they must be moved into a separate directory. Assuming this directory is `C:\oe_nuget\`, then the installation procedure is as followed:
```

Comment:
Can it source current directory with `.` instead or is an absolute path necessary?

Reply:
I reworded this to imply that any directory that contains the `.nupkg` files are okay, including the current one.

Reply:
Great, that is more clear. Perhaps also add a prior example command that shows that the nupkg files were moved to there to be even more clear.